src/components/chart/state-history-chart-line.ts

```diff
@@ -729,20 +729,17 @@ export class StateHistoryChartLine extends LitElement {
   }
 
   private _formatYAxisLabel = (value: number) => {
-    const formatOptions =
-      value >= 1 || value <= -1
```

Comment:
Can you explain why this is no longer needed? the comment still implies we only want to do it for tiny values?

Reply:
Yes but it should be if the delta with the previous value is tiny. I think I added that check before I started tracking the delta and only had access to the current value being formatted. So it was a bad heuristic that I should've removed

Reply:
Also checked with normal big numbers and setting `maximumFractionDigits` doesn't seem to affect the format for round numbers.
I guess the only effect of removing this check is in cases like #25836 it would display that number as 28.52 because we set `maximumFractionDigits` to 2+ . But that issue is fixed in another PR so we shouldn't get that case anymore